pkg/machinepool/machinepool_test.go

```diff
@@ -550,9 +550,16 @@ var _ = Describe("Utility Functions", func() {
 
 	Describe("minReplicaValidator function", func() {
 		var validator interactive.Validator
+		var replicaSizeValidationArgs *ReplicaSizeValidationArgs
 
 		BeforeEach(func() {
-			validator = minReplicaValidator(true, false, false) // or false for non-multiAZ
+			replicaSizeValidationArgs = &ReplicaSizeValidationArgs{
+				ClusterVersion: "openshift-v4.14.14",
+				MultiAz:        true,
+				IsHostedCp:     false,
+				Autoscaling:    false,
+			}
+			validator = minReplicaValidator(replicaSizeValidationArgs) // or false for non-multiAZ
```

Comment:
I very much welcome this change haha, `minReplicaValidator(true, false, false)` is not a pretty function call..